components/Aduan/Milestone.vue

```diff
@@ -501,6 +551,28 @@ export default {
               getNameStatus: dataStatusMilestone.dialihkan.getNameStatus
             }
           }
+        case dataStatusMilestone.gagalDiverifikasi.status:
+          return {
+            status: dataStatusMilestone.gagalDiverifikasi.textStatus,
+            icon: dataStatusMilestone.gagalDiverifikasi.icon,
+            fillColor: dataStatusMilestone.gagalDiverifikasi.fillColor,
+            getNameStatus: dataStatusMilestone.gagalDiverifikasi.getNameStatus
+          }
+        case dataStatusMilestone.pengerjaanDitunda.status:
+          return {
+            status: dataStatusMilestone.pengerjaanDitunda.textStatus,
+            icon: dataStatusMilestone.pengerjaanDitunda.icon,
+            fillColor: dataStatusMilestone.pengerjaanDitunda.fillColor,
+            getNameStatus: dataStatusMilestone.pengerjaanDitunda.getNameStatus
+          }
+        case dataStatusMilestone.pengerjaanDitinjauUlang.status:
+          return {
+            status: dataStatusMilestone.pengerjaanDitinjauUlang.textStatus,
+            icon: dataStatusMilestone.pengerjaanDitinjauUlang.icon,
+            fillColor: dataStatusMilestone.pengerjaanDitinjauUlang.fillColor,
+            getNameStatus:
+              dataStatusMilestone.pengerjaanDitinjauUlang.getNameStatus
+          }
```

Comment:
hello @rayfajars this `switch-case `catch my attention, usually `switch-case` block using string as a comparator like this:

```js
case 'GAGAL_DIVERIFIKASI':
case 'PENGERJAAN_DITUNDA':
...etc
```

why do you need to bind it to a variable, I'm curious about what the data looks like 🤔 

IMHO this switch-case scenario is hard to understand maybe we can simplify them

Reply:
Let's discuss this tomorrow, bli @Ibwedagama ?

I'm having a little trouble explaining it here

Reply:
I tryto make this code easier to read, i hope this can help 

[here](https://github.com/jabardigitalservice/super-app-web/pull/63/commits/8271722c8d91156db8568401bcd5914fba68935f)